03LessonThree/homework/git-scm.html

```diff
@@ -52,9 +53,9 @@
           </aside>
         </div>
     </main>
-  </footer>
+  <footer>
         This open sourced site is hosted on GitHub.<br/> Patches, suggestions and comments are welcome.
-    </div>
+    </footer>
 </body>
 
 </html>

```

Comment:
You closed all of the remaining semantic tags. Make sure you focus on indentation to improve code readability on the next assignment.